.github/workflows/documentation.yml

```diff
@@ -0,0 +1,50 @@
+name: documentation
+
+on:
+  pull_request:
+    branches: [documentation]
+  push:
+    branches: [documentation]
+
+jobs:
+ checks:
```

Comment:
Those don't seem to be checks. I would call that `build`.

Reply:
Semantically it checks if it builds. More checks would hopefully be added in the future.

Reply:
Hmm, in that case, my suggestion would be to separate this into two pipelines. You could also give it a speaking name for the current version and change it to checks later on if actual checks are added. I don't have a strong opinion here, just saying that it might be confusing.